hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/TenantStatsManagement.java

```diff
@@ -16,19 +16,17 @@
  * Management service for statistics of a single tenant.
  *
  */
-@FunctionalInterface
 public interface TenantStatsManagement {
 
     /**
-     * Service for stats of a single tenant. Opens a new transaction and as a
-     * result can an be used for multiple tenants, i.e. to allow in one session
-     * to collect data of all tenants in the system.
+     * Service for stats of a single tenant.
      *
-     * @param tenant
-     *            to collect for
      * @return collected statistics
      */
-    @PreAuthorize(SpringEvalExpressions.HAS_AUTH_SYSTEM_ADMIN)
-    TenantUsage getStatsOfTenant(String tenant);
+    @PreAuthorize(SpringEvalExpressions.HAS_AUTH_READ_REPOSITORY + SpringEvalExpressions.HAS_AUTH_OR
```

Comment:
I think there is no reason why only the system admin should have access to the stats. You can retrieve the data with the other permissions manually as well, so there is no need to hide the simplifying stats method.